Dockerfile

```
@@ -12,10 +12,12 @@ RUN groupadd -o -g 10941 squid && \
     useradd -o -u 10941 -g 10941 -s /sbin/nologin -d /var/lib/squid squid && \
     yum update -y && \
     yum install -y frontier-squid --enablerepo=osg-testing && \
-    rm -rf /var/cache/yum/*
+    rm -rf /var/cache/yum/* && \
+    mkdir /etc/squid/customize.d
```

```suggestion
    rm -rf /var/cache/yum/*
    
RUN mkdir /etc/squid/customize.d
```

It's nice to have this in another layer

---

I thought layers were bad because they cause an extra layer of overlayfs.  What is the benefit of having a layer just for that?

---

Ah I didn't think about that. I usually prefer the extra layers during development to make use of the cache but it probably makes sense to join things together in the finished product!